extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/LightStateConverter.java

```diff
@@ -164,9 +166,16 @@ public static int toAdjustedColorTemp(IncreaseDecreaseType type, int currentColo
      * the color temperature.
      *
      * @param lightState light state
-     * @return percent type representing the color temperature
+     * @return percent type representing the color temperature or null
+     *         if the bulb's {@link ColorMode} denotes the value is not applicable for setting color temperature
+     *         (i.e. the bulb's color mode is not {@link ColorMode.CT}).
      */
-    public static PercentType toColorTemperaturePercentType(State lightState) {
+    public static @Nullable PercentType toColorTemperaturePercentType(State lightState) {
+        ColorMode colorMode = lightState.getColorMode();
+        if (colorMode != null && !colorMode.equals(ColorMode.CT)) {
+            return null;
```

Comment:
Returning Java `null` here won't work - what you have to do is to send a channel state update `UnDefType.NULL`. Imho the best place to do the colorMode check would be [here](https://github.com/eclipse/smarthome/blob/65ca1fb1f6c4a2174cf2504ce8ac0eae3661f22c/extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java#L404) - simply do the update with PercentType it colorMode!=CT and otherwise do an `updateState(CHANNEL_COLORTEMPERATURE, UnDefType.NULL);`.
Makes sense?